packages/core/src/Stores/client-store.js

```diff
@@ -1834,6 +1834,26 @@ export default class ClientStore extends BaseStore {
     }
 
     setAccounts(accounts) {
+        // TODO: delete this code when BE will be fixed
```

Comment:
@sergei-deriv I don't think we should continue patching BE in FE anymore, Especially doing it here in the client store which will affect the whole codebase. Let's discuss this and find a better solution 🙏🏻 

cc: @yashim-deriv

Reply:
@farzin-deriv Agree with you) Kirill asked and I've tried to do this, but, of course, it's not a good solution and we have to avoid this approach. In this case I'll delete this code. @yashim-deriv ???